### PR TITLE
refactor: navbar dropdown width when showing dark mode option

### PR DESCRIPTION
**Pull request by alexbarnsley**
## Summary

https://app.clickup.com/t/1z9wt5a

## Checklist

-   [x] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [x] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I provided a screenshot of my changes to the component _(if applicable)_
-   [ ] I regenerated the `icons.html` file and checked if my newly added icon is shown correctly _(if necessary)_
-   [ ] I added an explanation on how to use the component to the readme _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

File: resources/views/navbar/profile.blade.php [hunk 1]
```diff
@@ -1,6 +1,13 @@
+@php($showDarkMode = config('ui.dark-mode.enabled') === true && ($profileMenuDarkMode ?? false))
+
 <x-ark-dropdown
     wrapper-class="whitespace-nowrap md:relative"
-    :dropdown-classes="'w-full md:w-auto mt-4 '.($profileMenuClass ?? null)"
+    :dropdown-classes="Arr::toCssClasses([
+        'mt-4',
+        'w-full md:w-auto' => ! $showDarkMode,
+        'w-64'             => $showDarkMode,
```
Comment (ItsANameToo) [inline resources/views/navbar/profile.blade.php:8]: do we need a width or a min-width? 🤔

Comment (alexbarnsley) [reply resources/views/navbar/profile.blade.php:8]: :thinking: min-width might be more appropriate for here yeh. Will update :+1:

**Comment by chriship** [comment]
I guess this is correct because you have specifically set it but I get different widths for the dropdown depending on whether dark mode is enabled.

Screenshot 2022-01-10 at 12 19 49

Screenshot 2022-01-10 at 12 20 03

**Comment by alexbarnsley** [comment]
Good catch too @chriship, thanks!

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**